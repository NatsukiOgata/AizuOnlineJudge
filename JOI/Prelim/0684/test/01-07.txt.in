49 51
69 73 57 95 75 97 59 18 84 72 10 21 98 28 40 16 8 23 29 41 5 34 66 71 46 87 64 22 36 6 80 37 100 74 17 82 26 14 96 56 47 45 51 48 49 65 68 67 79
88 9 20 94 49 67 55 6 1 44 81 35 96 58 26 31 100 97 28 85 56 4 47 19 8 21 91 37 30 90 29 80 95 93 40 41 86 24 82 65 5 64 61 34 22 3 68 17 13 11 27
