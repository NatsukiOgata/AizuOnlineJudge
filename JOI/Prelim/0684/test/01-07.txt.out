5
6
8
17
21
22
26
28
29
34
37
40
41
47
49
56
64
65
67
68
80
82
95
96
97
100
